packages/ui/Control/style.css.ts

```diff
@@ -0,0 +1,128 @@
+import { style, styleVariants } from '@vanilla-extract/css';
+import theme from "../theme.css";
+
+// Control 컴포넌트에서 사용되는 공통 스타일링
+export const controlWrapper = style({
+    display:'flex',
+    alignItems:'center',
+    cursor:'pointer',
+});
+
+export const controlLabel = styleVariants({
+    small: [
+        theme.fontsObject.BODY_3_14_R,
+        {
+          marginLeft: '6px',
+        },
+      ],
+      large: [
+        theme.fontsObject.BODY_2_16_R,
+        {
+          marginLeft: '8px',
+        },
+      ],
+});
+
+export const labelColor = styleVariants({
+    white: {
+      color: theme.colors.gray10,
+    },
+  });
+
+  // Radio 관련 스타일링
+  const radioBase = style({
+    all: 'unset',
+    borderRadius: '9999px',
+    border: `1.5px solid ${theme.colors.gray500}`,
+
+  }); 
+
+  export const radio = styleVariants({
+    small:[radioBase,{
+      width: '13px',
+      height: '13px',
+      selectors: {
+      '&:checked': {
+        width: '8px',
+        height: '8px',
+        border: `4px solid ${theme.colors.blue400}`,
+        backgroundColor: theme.colors.white,
+      },
```

Comment:
왜 width가 13인고 checked 되었을 때는 8인지에 대해 의문이 들었습니다. border 때문이었더라고요? 근데 그렇게 하려면 매번 border에 맞춰 계산을 해줘야 하는 불편함이 있을 거 같아서요.

```suggestion
  const radioBase = style({
    all: 'unset',
    borderRadius: '9999px',
    border: `1.5px solid ${theme.colors.gray500}`,
    boxSizing: 'border-box', // 추가
  }); 

  export const radio = styleVariants({
    small:[radioBase,{
      width: '16px', // 수정
      height: '16px', // 수정
      selectors: {
      '&:checked': {
        // width: '8px',
        // height: '8px',
        border: `4px solid ${theme.colors.blue400}`,
        backgroundColor: theme.colors.white,
      },
```

위와 같이 해주는 것이 더 추후에 수정할 때도 더 편할 거 같다는 생각인데 어떻게 생각하시나요?

(+) 토글의 
```ts
export const toggleWrapper = styleVariants({
  large: [toggleBase, { width: 32, height: 16, padding: 4 }],
  small: [toggleBase, { width: 24, height: 12, padding: 3 }],
});
```
이 부분도 마찬가집니다!

Reply:
아 맞네요 해당 속성 아예 생각도 못하고 있었네요..!

Reply:
[c2cdbc8](https://github.com/sopt-makers/makers-frontend/pull/98/commits/c2cdbc8e3e3a3e0ff40180ab45661688ce58b2f9)
[6aaf4bb](https://github.com/sopt-makers/makers-frontend/pull/98/commits/6aaf4bb488ce2ff0b646f109e3dd87d4a76e6761)

두 개 커밋에서 수정 완료했습니다!